tests/e2e/__testers__.ts

```diff
@@ -4,29 +4,48 @@ import { values } from 'lodash'
 import stripAnsi from 'strip-ansi'
 import { ClientBase } from '@prisma-spectrum/reflector/dist-cjs/Client'
 import { Reflector } from '@prisma-spectrum/reflector'
+import { getPrismaClient } from '~/tests/e2e/helpers/getPostgresAdminPrismaClient'
+import { PrismaClient } from '@prisma/client'
 
-export const testTemplate = (params: {
+export interface DBTestParams {
   templateName: PrismaTemplates.$Types.Template['_tag']
-  /** A string or regex pattern of the expected output (stdout) when the project dev mode is run.*/
   expectedDevOutput: RegExp | string
-}) => {
-  jest.setTimeout(50_000)
+  testDBURI?: string
+  datasourceProvider?: Reflector.Schema.DatasourceProviderNormalized
+  databaseUrlBase: string
+  getPrismaAdmin: getPrismaClient
+  dbLifeCycleEvents: {
+    resetDatabase: (prismaClient: PrismaClient, databaseName: string) => Promise<void>
+    initDatabase: (prismaClient: PrismaClient, databaseName: string) => Promise<void>
+  }
```

Comment:
Object says lifecycle events but the keys are verbs. Rethink the  identifier `dbLifeCycleEvents`. Simply `databaseActions`?